### DISCUSSION TITLE
Nemo : Add checkboxes in list view to select multiple files/folders

### DISCUSSION THREAD

**Discussion by pzim-devdata** [linuxmint/discussions#227] [Pull requests]
As a Nemo user, I would find it very useful to have checkboxes enabled in list view. This would allow for selecting and operating on multiple files and folders at once with more flexibility. 

This option could be an option to activate in Préférences > Display > Selection > Show checkboxes for instance 

It would be so great if  a Nemo developer would consider adding this functionality. 

Thanks for maintaining this excellent file manager!

**Comment by JosephMcc** [comment]
Don't think this would be very high priority since multi-selection can already be easily accomplished by using the Control and Shift key. It sort of falls in the "additional code complication for no real gain" category.

**Reply by pzim-devdata** [reply]
For me it's important because I use my mousse without keyboard on a media center. ;-)

**Reply by anaximeno** [reply]
It could be a new nemo action (actually a nemo-extension is more adequate) if someone wants to make that when the module it's ready. It will be exciting to see what people might come up with.
